buildtest/utils/file.py

```diff
@@ -57,6 +57,28 @@ def is_dir(dirname):
     return os.path.isdir(dirname)
 
 
+def is_symlink(sym_link):
+    """Check if the given link is a symlink and return ``True``, if it is not a symlink or the symlink is broken return ``False``.
+
+    Args:
+        sym_link (str): input link to check.
```

Comment:
Could change this to `filename (str): Input filename to check if its a symbolic link`